From 42b47145b961e8bdfc2aad5ae6baa55f57d2a029 Mon Sep 17 00:00:00 2001
From: ngam <67342040+ngam@users.noreply.github.com>
Date: Mon, 7 Nov 2022 00:06:11 -0500
Subject: [PATCH] change getargspec to getfullargspec

---
 pyclaw/src/pyclaw/solver.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyclaw/src/pyclaw/solver.py b/pyclaw/src/pyclaw/solver.py
index a67b45de..54266139 100644
--- a/pyclaw/src/pyclaw/solver.py
+++ b/pyclaw/src/pyclaw/solver.py
@@ -326,7 +326,7 @@ def _allocate_bc_arrays(self,state):
         import inspect
         for fun in (self.user_bc_lower,self.user_bc_upper,self.user_aux_bc_lower,self.user_aux_bc_upper):
             if fun is not None:
-                args = inspect.getargspec(fun)[0]
+                args = inspect.getfullargspec(fun)[0]
                 if len(args) == 5:
                     self.logger.warn("""The custom boundary condition
                                         function signature has been changed.
